src/webview/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -180,6 +183,25 @@ protected void onStart() {
 					default:
 						trace(this, "onActivityResult() :: no handling for requestCode=%s", requestCode);
 				}
+			} else if(requestCode == DISCLOSURE_LOCATION_PERMISSION_REQUEST) {
+				// User accepted or denied to allow the app to access
+				// location data in RequestPermissionActivity
+				if (resultCode == RESULT_OK) {					// user accepted
+					// Request to Android location data access
+					ActivityCompat.requestPermissions(
+							this,
+							LOCATION_PERMISSIONS,
+							ACCESS_FINE_LOCATION_PERMISSION_REQUEST);
+				} else if (resultCode == RESULT_CANCELED) {		// user rejected
+					try {
+						this.locationRequestResolved();
+						settings.setUserDeniedGeolocation();
+					} catch (SettingsException e) {
+						error(e, "Error recording negative to access location");
+					} catch (Exception e) {
```

Comment:
It's bad form to catch all Exceptions and I don't think it's required here. Can you remove this block?

Reply:
Removed.